packages/compass-sidebar/test/setup.js

```diff
@@ -1,15 +0,0 @@
-const hadronApp = require('hadron-app');
```

Comment:
This file and the ones in test/mocks as well, they didn’t seem to actually be consumed anywhere so I just removed them